python3 main.py > check.log 2> err.log
